WinUIGallery/Helper/WindowHelper.cs

```diff
@@ -31,7 +31,7 @@ static public Window CreateWindow()
         {
             Window newWindow = new Window
             {
-                SystemBackdrop = new MicaBackdrop()
+                //SystemBackdrop = new MicaBackdrop()
```

Comment:
Why?

Reply:
The capture API doesn't capture Mica, so there was a visual glitch when it swapped in the capture to run the effect.